pyccel/ast/itertoolsext.py

```diff
@@ -0,0 +1,21 @@
+
+from sympy import Tuple
+from .basic     import Basic
+
+__all__ = (
+    'Product',
+)
+
+class Product(Basic):
+    """Represents a Product stmt."""
```

Comment:
This comment could be clearer. It sounds like this represents a product statements (i.e. `a*b`) but I don't think this is the case right?

Reply:
Yes. poor description.
what about now?